File: JAVASCRIPT/3055.js
function mediaNotas(nota1, nota2) {
    return (nota1 + nota2) / 2;
  }
  
  function notaNaoSaber(nota1, media) {
    return 2 * media - nota1;
  }
  
  function main() {
    nota1 = parseInt(input());
    media = parseInt(input());
    nota2 = notaNaoSaber(nota1, media);
    console.log(nota2);
  }
  
  if (typeof main === "function") {
    main();
  }  

File: PYTHON/3055.py
def media_notas(nota_1, nota_2):
  return (nota_1 + nota_2) / 2


def nota_nao_saber(nota_1, media):
  return 2 * media - nota_1


def main():
  nota_1 = int(input())
  media = int(input())
  nota_2 = nota_nao_saber(nota_1, media)
  print(nota_2)


if __name__ == "__main__":
  main()
